<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageprocessing</class>
 <widget class="QWidget" name="imageprocessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>1149</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(254, 254, 254)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pBtestReport">
          <property name="text">
           <string>生成检测报告</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBimagesProcess">
          <property name="text">
           <string>超声图像分析</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1">
          <widget class="QPushButton" name="pBsetImagePath">
           <property name="text">
            <string>设定处理图像路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>95</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>处理的图像：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" rowspan="2" colspan="2">
          <widget class="QLabel" name="lbprocessImage">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="3">
          <widget class="QPushButton" name="pbSetRoi">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100)</string>
           </property>
           <property name="text">
            <string>选择区域</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pbCleanRoi">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100)</string>
           </property>
           <property name="text">
            <string>清理区域</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pbSetRegion">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100)</string>
           </property>
           <property name="text">
            <string>配置区域</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="5" colspan="4">
       <widget class="QGraphicsView" name="gVimageprocess">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(234, 234, 234)</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>gVimageprocess</zorder>
     <zorder>frame_3</zorder>
     <zorder>frame_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
